xcm/src/v3/mod.rs

```diff
@@ -662,6 +660,7 @@ pub enum Instruction<Call> {
 	/// Kind: *Instruction*
 	///
 	/// Errors: *Fallible*.
+	#[weight_args()]
```

Comment:
why do you want to bypass the weight args here?

Reply:
Because the weight function for this instruction doesn't require any arguments, so it's left empty.

Reply:
I don't think this is the right pattern. The weight function should have all parameters matching the instruction. If they are not used, they are just not used, but the instruction logic could potentially change in the future, using one of these parameters, and we dont want to change the function signature.